web/console/src/api/divisions.ts

```diff
@@ -49,4 +49,18 @@ export class DivisionsClient extends APIClient {
             responseData.statistics
         );
     }
+    /** gets division seasons reward status */
+    public async getSeasonsRewardStatus(): Promise<any> {
```

Comment:
not 'gets'...'Requests' sounds better.
also i guess prefix get not need here, just seasonRewardStatus(). And why any?